### ISSUE TITLE
Using an extension with a VPN

### ISSUE THREAD

**Issue by d3sance** [solidet-com/steam-currency-converter#7]
I'm having difficulty connecting to some resources on the internet, and I have to use a VPN. Your extension is great, but in order for it to work, I have to turn off the VPN, the VPN client allows me to tunnel traffic so that only certain sites pass through the VPN connection, adding the steam site to the tunneling, the extension still does not work, apparently I need to enter something else in order for the extension to work with the VPN.

**Comment by Trsnaqe** [comment]
Hi! I'm glad to hear you find the extension helpful. 

Could you let me know which country you switch to using the VPN? Also, could you try changing the base currency from the extension dropdown? The issue is likely not related to the connection but rather to the currency change between countries. A video with the browser console open would be really helpful for identifying the issue, too.

**Comment by d3sance** [comment]
The server is located in Finland, Helsinki. Here is a video with the console open https://imgur.com/a/xO5L6FU

**Comment by Trsnaqe** [comment]
Yup, as long as you are logged in, Steam continues to use Kazakhstan prices even if you change your location. Therefore, the extension keeps using Kazakhstan settings to match that behavior. If you try the same process without logging in, it should switch to Finland along with the Steam store.

Does Steam still use Kazakhstan prices after you switch to Finland while logged in? If so, the extension should be working as intended in this case. Could you show the Steam store after changing your VPN and provide a more detailed explanation of the issue? Thank you for collaborating; I’d love to help resolve this!

**Comment by d3sance** [comment]
I will describe the problem in detail: after turning on the VPN, the price conversion itself does not work on the game pages. I also noticed a feature after installing the extension with and without VPN. The welcome menu appears only when installing without VPN, otherwise, when VPN is enabled, this greeting does not appear, and prices are not converted.

The extension menu after the VPN is enabled, and the welcome menu that I was talking about
![{B2871155-034E-46C7-AE16-4D1A92EDA67F}](https://github.com/user-attachments/assets/6e1cbc7e-2339-4389-a131-700c9c79f49d) 
![{D0499FDA-450E-43B1-A140-C05399D7C9C0}](https://github.com/user-attachments/assets/10eee9b8-55a0-4990-8c6f-2e7ffb71a696)

**Comment by Trsnaqe** [comment]
Hey can you try again after the latest update please if it is resolved or not, thanks!

**Comment by d3sance** [comment]
The problem with using the VPN extension has not been resolved, but now I use other crawls for websites, and the extension works fine ;)

**Comment by Trsnaqe** [comment]
I see, I am glad that you are able to use extension. However since the issue havent been resolved yet I am not going to close it in case that someone would be able to replicate and provide help on this. Thanks for your feedback and sorry as we werent able to resolve the issue.
